aws-blog-event-driven-batch-analytics/State-Management-Store.md

```diff
@@ -0,0 +1,45 @@
+
+
+## State Management Store
+
+A simple statement management store can be accomplished with two tables: one for storing aggregation job configurations (AGGRJOBCONFIGURATION) and another for storing the ingested file status (INGESTEDFILESTATUS). Here are the columns in each table and their descriptions. The DDL and the EMR job configurations for the tracking store can be found here
+
+Sample records for the AGGRJOBCONFIGURATION are also provided here as a reference. They indicate that the job for Illinois state will not be submitted unless there are alteast 13 vendor transaction files are posted and master data for Item data is posted. Similar rules were also configured for the state of California with the execption of minimum file count is 25.
+
+
+
+### AGGRJOBCONFIGURATION
```

Comment:
Shall we have the datatype also included, just to enhance the customer's understanding?